crates/uv/tests/tool_install.rs

```diff
@@ -3007,3 +3007,82 @@ fn tool_install_at_latest_upgrade() {
         "###);
     });
 }
+
+/// Test installing a tool together with some additional entrypoints
+/// from other packages.
+#[test]
+fn tool_install_additional_entrypoints() {
+    let context = TestContext::new("3.12")
+        .with_filtered_counts()
+        .with_filtered_exe_suffix();
+    let tool_dir = context.temp_dir.child("tools");
+    let bin_dir = context.temp_dir.child("bin");
+
+    uv_snapshot!(context.filters(), context.tool_install()
+        .arg("--i-want-ponies")
+        .arg("ansible-core")
+        .arg("--i-want-ponies")
+        .arg("ansible-lint")
+        .arg("ansible==9.3.0")
```

Comment:
Minor note: This is probably an expensive test because the package is big. We might want to find a smaller package to keep the test case minimal.

Reply:
Many dependencies were coming from `ansible-lint`, which is an extra package providing an additional entrypoint. I replaced that with `black`, as it covers the same scenario with fewer transitive deps.

If you have an idea about a better combo than `ansible` and `ansible-core`, I'll happily replace them here.